elements/tuto-reader/tutorials/alternate-firmware/docs/installing-alternate-firmware.md

```diff
@@ -1 +1,183 @@
+approach helps achieve minimal boot times and a small footprint.
+
+With the separation of hardware initialization and later boot logic, Coreboot can do 
+the following: 
+
+- Scale from specialized applications that run directly from firmware 
+- Run operating systems in flash 
+- Load custom boot-loaders 
+- Implement firmware standards such as PC BIOS services or UEFI 
+
+By permitting systems the ability to include only the features necessary to run the 
+target application, code and flash space required are reduced. For further information, 
+refer to the [Coreboot website]( http://www.coreboot.org).
+
+The status on Coreboot's [Supported Motherboards](http://www.coreboot.org/Supported_Motherboards) 
+page indicates an unknown status for MinnowBoard boards; however, mainline coreboot 
+support for MinnowBoard boards (by Sage Electronic Engineering) does exist, as shown 
+on [review.coreboot.org](http://review.coreboot.org/gitweb?p=coreboot.git). Coreboot 
+supports booting from Serial ATA and USB 2.0/3.0. Booting from SD storage isn't working, 
+and S3 suspend/resume support isn't working either. 
+
+Below is a step-by-step guide to building coreboot for the Minnowboard MAX and Turbot.
+
+#### Install the dependencies
+
+  ``` 
+  $ sudo apt-get install gcc git make ncurses-dev flex bison
+  ```
+
+#### Get Coreboot 
+For simplicity, put all downloads and items extracted into the same directory.
+
+  ```
+ 	$ git clone http://review.coreboot.org/p/coreboot
+ 	$ cd coreboot
+ 	$ git submodule update --init --checkout
+ 	$ git checkout b9a0809faeeef67e46cda17cf8f7a839c6fe614c
+ 	$ wget https://minnowboard.org/tutorials/alternate-firmware/docs/Minnowboard_Max_Coreboot_config.txt
+  ```
+
+#### Get Intel® Firmware Support Package (Intel® FSP)
+1. Download either a Windows or Linux version, depending on the needs of your system:
+    1. [Windows](http://www.intel.com/content/www/us/en/embedded/software/fsp/atom-e3800-fsp-g3-windows-download.html){n}
+    1. [Linux](http://www.intel.com/content/www/us/en/embedded/software/fsp/atom-e3800-fsp-g3-linux-download.html){n}
+    1. Archive [Intel® FSP for Intel® Atom™ Processor E3800 Series](http://downloadcenter.intel.com/download/24496){n}
+2. To extract from archive, follow the instructions in the Readme_Extract.
+
+#### Get Intel® Binary Configuration Tool (Intel® BCT) 
+1. Download either a Windows or Linux version, depending on the needs of your system: 
+    1. [Windows](https://edc.intel.com/Link.aspx?id=10034){n}
+    1. [Linux 32-bit](https://edc.intel.com/Link.aspx?id=10033){n}  
+    1. [Linux 64-bit](https://edc.intel.com/Link.aspx?id=10032){n}
+
+#### Setup Firmware Support Package
+1. See [this guide](http://www.intel.com/content/www/us/en/embedded/products/bay-trail/atom-e3800-minnowboard-max-platform-guide.html) for the most current info. The following is from an older version previously tested to work. 
+
+  ```
+  $ cd bct
```

Comment:
I have two concerns with this content:

1. Separated chunks of terminal steps should be self-supported--meaning if the use opened a new terminal and typed what the shell steps are, it would work. In this example, where is the `bct` directory?

To correct, in the prior steps it should set a base reference directory, eg: `cd ~` and then in this section, the `cd bct` could be `cd ~/coreboot/bct`.

2. If we haven't tested the steps, we shouldn't be documenting them. Why haven't we tested with the newest version? What "older version" did these steps work with?